README.md

```diff
@@ -14,7 +14,10 @@ Vertx Extra
          alt="Maven Artifact">
 </a>
 
-Vertx compatible wrapper around the metrics Java client. 
+Vertx 3.x compatible wrapper around the metrics Java client. 
+
+For Vertx 2.x you must use the 0.4.x version of this library.  There will be no further development in support of Vertx 2.x and future Vertx 3.x integration 
```

Comment:
Nit-pick. Please use two different variables in the 2.x and 0.4.x version series identifiers since "x" is does not need to be the same across the two series.